dimod/quadratic/quadratic_model.py

```diff
@@ -493,6 +520,11 @@ def iter_neighborhood(self, v: Variable) -> Iterator[Tuple[Variable, Bias]]:
     def iter_quadratic(self) -> Iterator[Tuple[Variable, Variable, Bias]]:
         return self.data.iter_quadratic
 
+    @forwarding_method
+    def lower_bound(self, v: Variable) -> Bias:
```

Comment:
should this be `-> Integer` instead? (same for `upper_bound`)

Reply:
Yeah, it's weird, but no. For various performance reasons we store the bounds under the hood as `bias_type`, the same as bias.